composer.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "netgen/ibexa-scheduled-visibility",
```

Comment:
Four spaces in the `composer.json` file.

Reply:
fixed in c6fed8b415a0806f335cb67eaad3a1bce8cc5c7d